scripts/sync_readme_gallery.py

```
@@ -24,89 +24,88 @@ class GallerySection:
     ),
     GallerySection(
         heading="### 📚 Knowledge",
         summary="This is your team's memory: wiki pages, source material, and reusable context.",
-        screenshots=(Screenshot("docs/screenshots/readme/v3-knowledge.png", "Knowledge and Wiki"),),
+        screenshots=(Screenshot("docs/screenshots/readme/v4-knowledge.png", "Knowledge and Wiki"),),
     ),
     GallerySection(
         heading="### 🔭 Logs and activity",
-        summary="This helps you answer, ‘what just happened?’",
-        screenshots=(Screenshot("docs/screenshots/readme/v3-logs.png", "Logs"),),
+        summary="This helps you answer, ‘what just happened?’ — every LLM call, token count, latency, and cost.",
```

Ruff (0.15.13)

[warning] String contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)